File: 2264-minimum-sum-of-four-digit-number-after-splitting-digits/minimum-sum-of-four-digit-number-after-splitting-digits.py
class Solution:
    def minimumSum(self, num: int) -> int:
        i = 0
        nums = [0] * 4
        while num > 0:
            nums[i] = num % 10
            num //= 10
            i += 1
        
        nums.sort()
        num1 = nums[0]*10 + nums[2]
        num2 = nums[1]*10 + nums[3]

        return num1 + num2

        

File: 2264-minimum-sum-of-four-digit-number-after-splitting-digits/minimum-sum-of-four-digit-number-after-splitting-digits.java
class Solution {
    public int minimumSum(int num) {
        int nums[] = new int[4];
        int i = 0;
        while(num > 0) {
            nums[i++] = num % 10;
            num = num / 10;
        }
        Arrays.sort(nums);
        int num1 = nums[0] * 10 + nums[2];
        int num2 = nums[1] * 10 + nums[3];
        return num1 + num2;
    }
}